presto-tests/src/main/java/com/facebook/presto/tests/statistics/Metrics.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.statistics;
+
+import com.facebook.presto.cost.PlanNodeStatsEstimate;
+import com.facebook.presto.cost.SymbolStatsEstimate;
+
+import java.util.Optional;
+import java.util.OptionalDouble;
+
+import static java.lang.Double.isInfinite;
+import static java.lang.Double.isNaN;
+
+public final class Metrics
+{
+    private Metrics() {}
+
+    public static final Metric OUTPUT_ROW_COUNT = new Metric()
+    {
+        @Override
+        public OptionalDouble getValueFromPlanNodeEstimate(PlanNodeStatsEstimate planNodeStatsEstimate, StatsContext statsContext)
+        {
+            return asOptional(planNodeStatsEstimate.getOutputRowCount());
+        }
+
+        @Override
+        public OptionalDouble getValueFromAggregationQueryResult(Object value)
+        {
+            return OptionalDouble.of(((Number) value).doubleValue());
+        }
+
+        @Override
+        public String getComputingAggregationSql()
```

Comment:
just `getAggregationSql`?

Reply:
i will rename it to `getAggregationExpression`.

Reply:
`getAggregationSql` sounds better